docs/source/common_issues.rst

```diff
@@ -173,6 +173,82 @@ and difficult-to-predict failure modes and could result in very
 confusing error messages. The tradeoff is that you as a programmer
 sometimes have to give the type checker a little help.
 
+.. _version_and_platform_checks:
+
+Python version and system platform checks
+-----------------------------------------
+
+Mypy supports the ability to perform Python version checks and platform
+checks (e.g. Windows vs Posix), ignoring code paths that won't be run on
+the targeted Python version or platform. This allows you to more effectively
+typecheck code that supports multiple versions of Python or multiple operating
+systems.
+
+More specifically, mypy understands ``if/elif`` statements of the following
+forms:
```

Comment:
This looks very formal. The rest of this file is all written using examples (as is PEP 484 in this case). 

I would also pick examples that are practical: e.g. for sys.version_info, show distinguishing Python 2 vs. 3 and some distinction between e.g. 3.4 and higher. For sys.platform, I'd show `startswith('linux')`, `== 'darwin'`, and `== 'win32'`, since those cover the major platform. We're probably not going to spend much effort in supporting cygwin so I'd leave that out.